cmd/relayproxy/config/notifier.go

```diff
@@ -5,8 +5,8 @@ import "fmt"
 type NotifierConf struct {
 	Kind NotifierKind `mapstructure:"kind" koanf:"kind"`
 	// Deprecated: Use WebhookURL instead
-	SlackWebhookURL string              `mapstructure:"slackWebhookUrl" koanf:"slackwebhookurl"`
-	EndpointURL     string              `mapstructure:"endpointUrl" koanf:"endpointurl"`
+	SlackWebhookURL string              `mapstructure:"slackWebhookUrl" koanf:"slackWebhookUrl"`
+	EndpointURL     string              `mapstructure:"endpointUrl" koanf:"endpointUrl"`
```

Comment:
**🔨 issue:** koanf annotation expect a lower case name here, why have you changed it for slack and url endpoint? It has nothing to do with this PR.